app/views/layouts/application.html.rb

```diff
@@ -19,6 +19,8 @@ def content
 
         meta(property: "og:url", content: "http://www.superdelegation.com")
         meta(property: "og:image", content: 'http://i.imgur.com/vM4Kr5M.jpg')
+
+        render 'shared/analytics'
```

Comment:
Heap specifies that it loads in the `<head>` tag: https://heapanalytics.com/docs/installation#web